tests/test_sentry.py

```diff
@@ -131,7 +131,7 @@ def test_task():
         content_type='application/json',
         status=200,
     )
-    test_task()
+    test_task.delay()
```

Comment:
I don't think this should be changed - but i'm a bit surprised that the test pass :-)